scripts/superdesk/editor2/editor.js

```diff
@@ -1020,7 +1020,10 @@ angular.module('superdesk.editor2', [
                         scope.medium.subscribe(eventName, updateAddContentButton);
                     });
                     // listen updates by medium editor to update the model
-                    scope.medium.subscribe('editableInput', function() {
+                    scope.medium.subscribe('editableInput', function(e, elem) {
+                        elem.querySelectorAll('span[style]')
+                            .forEach(span => span.removeAttribute('style'));
```

Comment:
I don't think so, it's just added by chrome, in firefox it shoudn't be there at all